Makefile.include

```diff
@@ -532,7 +532,12 @@ ifeq (,$(FLASHFILE))
 endif
 
 # By default always build ELFFILE and FLASHFILE
-BUILD_FILES += $(ELFFILE) $(FLASHFILE)
+ifeq ($(RIOT_CI_BUILD),1)
+  BUILD_FILES += $(ELFFILE) $(FLASHFILE)
+else
+  # Also build BINFILE when not on CI
```

Comment:
I would move the comment in the block above: `Don't build BINFILE on the CI to save some computation time`, or something like this.

Reply:
Done.